Sources/Transaction/TxOutMemoBuilder.swift

```diff
@@ -44,41 +44,47 @@ class TxOutMemoBuilder {
     init(ptr: OpaquePointer) {
         self.ptr = ptr
     }
-    
+
     deinit {
         mc_memo_builder_free(ptr)
     }
 
     static func createRecoverableMemoBuilder(accountKey: AccountKey) -> RecoverableMemoBuilder {
         RecoverableMemoBuilder(accountKey: accountKey)
     }
-    
+
     static func createDefaultMemoBuilder() -> DefaultMemoBuilder {
         DefaultMemoBuilder()
     }
-    
-    static func createRecoverablePaymentRequestMemoBuilder(paymentRequestId: UInt64, accountKey: AccountKey) -> RecoverablePaymentRequestMemoBuilder {
-        RecoverablePaymentRequestMemoBuilder(paymentRequestId: paymentRequestId, accountKey: accountKey)
+
+    static func createRecoverablePaymentRequestMemoBuilder(
+            paymentRequestId: UInt64,
+            accountKey: AccountKey
```

Comment:
single indentation for function params in function definition
```suggestion
        paymentRequestId: UInt64,
        accountKey: AccountKey
```

Reply:
accepted